src/Packetery/Module/DashboardWidget.php

```diff
@@ -141,9 +140,9 @@ public function setup(): void {
 	 * @return bool
 	 */
 	private function isPacketaShippingMethodActive(): bool {
-		/** @var WC_Shipping_Zone_Data_Store $shippingDataStore */
 		$shippingDataStore = WC_Data_Store::load( 'shipping-zone' );
-		$shippingZones     = $shippingDataStore->get_zones();
+		/** @phpstan-ignore method.notFound */
```

Comment:
Předchozí řešení se mi líbí víc, v čem je problém?

Reply:
yes, moje blbost.. vyignoruji tu chbu se subtypem.

 PHPDoc tag @var with type WC_Shipping_Zone_Data_Store is not subtype of type WC_Data_Store. 
 
 Protože ta třídá WC_Shipping_Zone_Data_Store se vrací pomocí magie. :(